src/main/java/com/epam/repository/TourOfferDAO.java

```diff
@@ -17,24 +17,24 @@
 @Repository
 public class TourOfferDAO implements SimpleTourOfferDAO {
     private final JdbcTemplate JdbcTemplate;
-
     @Value("${tourOffer.tableName}")
     private String tableName;
+
     @Autowired
     public TourOfferDAO(JdbcTemplate JdbcTemplate) {
         this.JdbcTemplate = JdbcTemplate;
     }
 
     public List<TourOffer> getTours(){
-        return JdbcTemplate.query("SELECT * FROM " +  tableName, (rs, rowNum) -> buildTour(rs));
+        return JdbcTemplate.query("SELECT * FROM "+tableName, (rs, rowNum) -> buildTour(rs));
     }
 
     public int deleteTour(Integer tourId){
-         return JdbcTemplate.update("DELETE FROM " + tableName+" WHERE id = ?", tourId);
+         return JdbcTemplate.update("DELETE FROM "+tableName+" WHERE id = ?", tourId);
     }
 
     public int addTour(TourOffer touroffer){
-        return JdbcTemplate.update("INSERT INTO "+tableName+" (tour_type, start_date, end_date, price_per_unit, hotel_id, description, discount_id) " +
+        return JdbcTemplate.update("INSERT INTO "+tableName+"(tour_type, start_date, end_date, price_per_unit, hotel_id, description, discount_id) " +
```

Comment:
please, use property variables here and in all other methods

Reply:
Can i fix it in another PR?

Reply:
@VNTee @imbirevkotlya I think u can. Please resolve conflicts